.circleci/config.yml

```diff
@@ -53,27 +53,27 @@ jobs:
   # JOB: test
   # Test that the website can be built.
   # --------------------------------------------------
-  test:
-    executor: node
-    working_directory: ~/react-native-website/website
-    steps:
-      - restore_cache_checkout
-      - run_yarn
-      - run: yarn ci-check
-      - run: export NODE_OPTIONS=--max_old_space_size=4096
-      - run: export TERSER_PARALLEL=false
-      # Run build test only for three latest version to avoid memory issues
-      - run: export DEPOLY_PREVIEW=true
-      - run:
-          name: Compile website
-          command: yarn test
-          no_output_timeout: 20m
-      - run:
-          name: Check for missing index.html (build errors)
-          command: |
-            if [ ! -f build/index.html ]; then
-              exit 1;
-            fi
+#  test:
```

Comment:
Why comment it out instead of deleting it?

Reply:
I have commented out the `test` workflow temporarily for testing, will remove the code before the merge, thank for pointing this out! :)